index.html

```diff
@@ -10,6 +10,27 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    
+    <section class="hero">
+        <div class="conatiner flexContainer">
+            <header class="heroHeader">
+                <p>Welcome to</p>
+                <h1>Family Bakery</h1>
+                <p>
+                    We're the biggest, best equipped and most advanced Bakery in the greater Los Angeles area.
+                </p>
+            </header>
+            <div class="setting">
```

Comment:
icons are not showing 
![image](https://user-images.githubusercontent.com/44810632/155527227-50b9fd77-757b-4b9f-a54e-95d6027fe24d.png)

Reply:
same "font family issue" i added front awsome and didn't pull from dev

Reply:
same "font family issue" i added front awsome and didn't pull from dev

Reply:
same "font family issue" i added front awsome and didn't pull from dev